### ISSUE TITLE
Pagination with query `select(class)` results in wrong serialization

### ISSUE THREAD

**Issue by mmzeynalli** [uriyyo/fastapi-pagination#1312]
So, if I have an SQLALchemy model:

```python
from fastapi_pagination.ext.sqlalchemy import paginate
from fastapi_pagination.links import Page

class Skill(Base):
  name: str
  created_at: datetime

class CoreDataSchema(BaseModel):
    name: str

@router.get('/search', response_model=Page[CoreDataSchema])
async def get_and_search_core_models(db: DbDependency):
    query = select(Skill)
    return paginate(db, query)
```

This returns error in validation, as the result of query is Row(SkillObject,), which is not being seiralized by `CoreDataSchema`. This is the result of the last version, `0.12.28` where line was added at `sqlachemy.py:212`

```python
@no_type_check
def _should_unwrap_scalars(query: Selectable) -> bool:
    try:
        return len(query.column_descriptions) == 1 and len(query.columns) > 1
    except AttributeError:
        return True

# The rest of code

if _should_unwrap_scalars(query):  # line 212
        items = unwrap_scalars(items)
```

As Skill has two columns, `len(query.columns) > 1` returns false, ultimately never "scalaring" the resulting rows.  However, if I do `select(Skill.name)`, it works just fine

**Comment by uriyyo** [comment]
Hi @mmzeynalli,

Could you please provide code snippet and library version that you are using so I can reproduce issue on my side?

**Comment by mmzeynalli** [comment]
I literally did that in my first comment.

Version: 0.12.28 (0.12.27 works)

Code:

```python
from fastapi_pagination.ext.sqlalchemy import paginate
from fastapi_pagination.links import Page

class Skill(Base):
  name: str
  created_at: datetime

class CoreDataSchema(BaseModel):
    name: str

@router.get('/search', response_model=Page[CoreDataSchema])
async def get_and_search_core_models(db: DbDependency):
    query = select(Skill)
    return paginate(db, query)
```

**Comment by vahidzhe** [comment]
I have some problem

**Comment by uriyyo** [comment]
@mmzeynalli Unfortunately, I can't reproduce this issue with code snippet that you give me.

Could you please provide me with code that I can copy-paste and run?

**Comment by mmzeynalli** [comment]
After throughout research, I understood that problem lies elsewhere: the problem occurs IF and ONLY IF the model has one field:

Example Code:

```python
from typing import Annotated, Any, Generator

from fastapi import APIRouter, Depends, FastAPI
from fastapi_pagination import add_pagination
from fastapi_pagination.ext.sqlalchemy import paginate
from fastapi_pagination.links import Page
from pydantic import BaseModel
from sqlalchemy import MetaData, create_engine, select
from sqlalchemy.orm import DeclarativeBase, Mapped, Session, mapped_column, sessionmaker

from fromfolio.config import settings

# create endgine and check database
engine = create_engine(settings.DATABASE_URL)

SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)


class Base(DeclarativeBase):
    metadata = MetaData()


def get_db() -> Generator[Session, Any, None]:
    """This function serves as a context manager to manage the database session"""
    db = SessionLocal()
    try:
        yield db
    finally:
        db.close()


DbDependency = Annotated[Session, Depends(get_db)]


app = FastAPI()
add_pagination(app)

router = APIRouter()


class Test(Base):
    __tablename__ = 'core_test'
    name: Mapped[str] = mapped_column(primary_key=True)

class CoreDataSchema(BaseModel):
    name: str


@router.get('/search', response_model=Page[CoreDataSchema])
async def get_test(db: DbDependency):
    query = select(Test)

    return paginate(db, query)

app.include_router(router)
```

**Comment by uriyyo** [comment]
@mmzeynalli Could you try to install latest version of `fastapi-pagination` and add `unwrap_mode="unwrap"`?

```py
@router.get('/search', response_model=Page[CoreDataSchema])
async def get_test(db: DbDependency):
    query = select(Test)

    return paginate(db, query, unwrap_mode="unwrap")
```

**Comment by mmzeynalli** [comment]
Yes, this works fine. But, I believe, it is not patch update, because, it forces me to change my code all over the project, so I think, it would be better to do minor or major version update.

**Comment by uriyyo** [comment]
@mmzeynalli My bad, I'm preparing `0.13.0` release  that will remove deprecated modules and wanted to fix this issue before actually releasing `0.13.0`.
